cla_frontend/templates/zendesk-feedback.txt

```diff
@@ -0,0 +1,13 @@
+--- Comments -----------------
+
+{{ comments|safe }}
```

Comment:
This is being sent as plain text, without safe filter Django will convert special characters (apostrophes, ampersands etc) into HTML entities.